ert_shared/storage/http_server.py

```diff
@@ -102,13 +106,30 @@ def response_by_name(self, ensemble_id, response_name):
             resolve_ref_uri(response, ensemble_id)
             return response
 
+    def get_data(self, ids):
+        def generator():
+            with StorageApi(
+                    rdb_url=self._rdb_url, blob_url=self._blob_url
+            ) as api:
+                first = True
+                for data in api.data(ids):
+                    if first:
+                        first = False
+                    else:
+                        yield "\n"
+                    if isinstance(data, list):
+                        yield ",".join([str(x) for x in data])
+                    else:
+                        yield str(data)
+        return Response(generator())
+
     def data(self, data_id):
-        with StorageApi(rdb_url=self._rdb_url, blob_url=self._blob_url) as api:
-            data = api.data(data_id)
-            if isinstance(data, list):
-                return ",".join([str(x) for x in data])
-            else:
-                return str(data)
+        return self.get_data([data_id])
+
+    def datas(self):
+        ids = request.args.getlist("id")
```

Comment:
Please use the body. There's usually a pretty small URL max length, so would probably crash if I ask for 50k datas (which should be possible).

Reply:
yes, I was thinking this PR is just for looking at. We should merge your approach and mine once we have compared and discussed

Reply:
Also, as we discussed your use of POST: I think that is necessary actually. Body in GET is allowed, but SHOULD not be used to define the result (allthough that too is allowed, but then we cannot expect cache-compatibility, and should probably label the response as uncacheable).
I think we could consider having both approaches. It could be quite useful to have urls representing the resource "all data for result B", and it makes client code much simpler. If we ever hit a url limit, then the other approach is probably necessary, but that doesn't take away the usefulness of plain old urls existing where possible.
Another solution is to design endpoints on the server that have a sort of query structure to them, where we can do something like:
http://server/dataquery?ensemble=1&response=2, where one can omit and include query params as needed to limit the result, which could support a host of different ways of querying, for different needs.